src/agent/Makefile.am

```diff
@@ -90,6 +95,15 @@ libotbr_agent_la_LIBADD                                       = \
     $(top_builddir)/src/utils/libutils.la                       \
     $(NULL)
 
+if OTBR_ENABLE_NCP_OPENTHREAD
+libotbr_agent_la_LIBADD                                      += \
```

Comment:
define `OTBR_ENABLE_OPENWRT` to guard this library together with `OTBR_ENABLE_NCP_OPENTHREAD`